### PR TITLE
Fix order of assets in tokenBundle

### PR DESCRIPTION
**Pull request by PeterBenc**

### CODE REVIEW

File: app/frontend/wallet/helpers/tokenFormater.ts [hunk 1]
```diff
@@ -33,11 +33,19 @@ export const aggregateTokenBundles = (tokenBundle: TokenBundle[]): TokenBundle =
     .flatten()
     .value()
 
-export const groupTokenBundleByPolicyId = (
-  tokenBundle: TokenBundle
-): {[policyId: string]: TokenBundle} => {
+export const groupTokenBundleByPolicyId = (tokenBundle: TokenBundle) => {
```
Comment (refi93) [inline app/frontend/wallet/helpers/tokenFormater.ts:36]: I'd specify the return type explicitly

File: app/frontend/wallet/shelley/shelley-transaction.ts [hunk 2]
```diff
@@ -120,7 +120,7 @@ function cborizeTxInputs(inputs: TxInput[]): CborizedTxInput[] {
 function cborizeTxOutputTokenBundle(tokenBundle: TokenBundle): CborizedTxTokenBundle {
   const policyIdMap = new Map<Buffer, Map<Buffer, number>>()
   const tokenObject = groupTokenBundleByPolicyId(tokenBundle)
```
Comment (refi93) [inline app/frontend/wallet/shelley/shelley-transaction.ts:122]: I'd rename this to `orderedTokenBundle` and `groupTokenBundleByPolicyId` to `toOrderedTokenBundle` and perhaps even have a OrderedTokenBundle type to make it clear when such bundle is dealt with in the logic

File: app/frontend/wallet/shelley/shelley-ledger-crypto-provider.ts [hunk 3]
```diff
@@ -208,7 +208,7 @@ const ShelleyLedgerCryptoProvider = async ({
       })
     }
     const tokenObject = groupTokenBundleByPolicyId(tokenBundle)
-    return Object.entries(tokenObject).map(([policyId, assets]) => {
+    return tokenObject.map(({policyId, assets}) => {
```
Comment (refi93) [inline app/frontend/wallet/shelley/shelley-ledger-crypto-provider.ts:211]: same renaming/typing remark as below

File: app/frontend/wallet/shelley/shelley-transaction.ts (hunk 2 above)
Comment (PeterBenc) [reply app/frontend/wallet/shelley/shelley-transaction.ts]: Great idea, fixed.

**Review by refi93 — APPROVED** [review]

**Pull request merged**